dns-records-office365/azuredeploy.json

```diff
@@ -0,0 +1,72 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"dnsZoneName": {
+			"type": "string",
+			"metadata": {
+				"description": "Name of the DNS Zone to create or update."
+			}
+		},
+		"ttl": {
+			"type": "int",
+			"defaultValue": 3600,
+			"metadata": {
+				"description": "Time to live value, in seconds"
+			}
+		},
+		"artifactsBaseUrl": {
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts